app/assets/stylesheets/hyrax/accessibility_tweaks.css

```diff
@@ -0,0 +1,4 @@
+table.related-files td.attribute-permission span.label {
```

Comment:
It might be a good pattern to have a comment line before a new CSS block indicating the story it's for?